tests/e2e-playwright/tests/sleepers/sleepers.py

```diff
@@ -70,10 +71,24 @@ def test_sleepers(
     create_new_project_and_delete(auto_delete=True)
 
     # we are now in the workbench
-    for _ in range(1, _NUM_SLEEPERS):
+    for _ in range(1, num_sleepers):
         page.get_by_text("New Node").click()
-        page.get_by_text("sleeperA service which").click()
-        page.get_by_text("Add", exact=True).click()
+        page.get_by_placeholder("Filter").click()
+        page.get_by_placeholder("Filter").fill("sleeper")
+        page.get_by_placeholder("Filter").press("Enter")
+
+    # set inputs if needed
+    if input_sleep_time:
+        for index, sleeper in enumerate(page.get_by_test_id("nodeTreeItem").all()[1:]):
+            print(f"---> setting sleeper {index} input time to {input_sleep_time}...")
+            sleeper.click()
+            sleep_interval_selector = page.get_by_role("textbox").nth(1)
+            sleep_interval_selector.click()
+            sleep_interval_selector.fill(f"{input_sleep_time}")
+            print(f"<--- sleeper {index} input time set to {input_sleep_time}")
+        workbench_selector = page.query_selector("#SvgjsSvg1001")
```

Comment:
Q: is this the selector of a resource? This ID looks autogenerated somehow.
Could we not give a stable css id to the root element in the HTML and use that one?

Reply:
This implies changing the JS code. so yes possible, I will check with @jsaq007 / @ignapas where to do that